src/app/public/modules/text-editor/text-editor.component.html

```diff
@@ -0,0 +1,42 @@
+<div
+  class="sky-text-editor"
+>
+  <sky-toolbar
+    class="sky-text-editor-toolbar"
+  >
+    <sky-toolbar-section
+      aria-hidden="true"
```

Comment:
Why are we removing the toolbars from a11y scope?

Reply:
This was also deliberate by the contributor. Let's talk offline.